framework/integration/client/init.go

```diff
@@ -127,11 +102,32 @@ func mustSetup() {
 	}
 }
 
-func mustTeardown() {
+func MustTeardown(namespace string) {
 	err := k8sClient.CoreV1().Namespaces().Delete(namespace, nil)
 	if errors.IsNotFound(err) {
 		// fall though
 	} else if err != nil {
 		panic(err)
 	}
 }
+
+func init() {
+	k8sClient, err = newK8sClient()
+	if err != nil {
+		panic(err)
+	}
+}
```

Comment:
I personally like to see init() on top (even though it's private). It's kind of magical ✨ function.

Reply:
Done.